src/it/resources/devnetTemplate.conf

```diff
@@ -55,6 +55,6 @@ scorex {
   }
   restApi {
     bindAddress = "0.0.0.0:9051"
-    apiKeyHash = null
+    apiKeyHash = "324dcf027dd4a30a932c441f365a25e86b173defa4b8e58948253471b81b72cf"
```

Comment:
This is related to https://github.com/ergoplatform/ergo/issues/1316 ... when all the configs get merged, `apiKeyHash`  eventually turns into empty string which is sort of a bug ... Anyway integration tests of wallet require authentication with api key so I added it to be able to authenticate

Reply:
Looks good